src/index.html

```diff
@@ -12,21 +12,13 @@
     />
   </head>
   <body>
-    <main>
+    <main class="min-vh-100 d-flex flex-column justify-content-center">
       <div class="container-md">
-        <h1>Leaderboard</h1>
-        <div class="row mx-0">
-          <section id="recent" class="col-6">
-            <span class="h2">Recent scores</span>
-            <button id="refresh-btn" type="button" class="btn btn-primary">Refresh</button>
-            <table class="table table-striped">
-              <tbody id="result-list"></tbody>
-            </table>
-            <div id="refresh-msg" class="text-center d-none opacity-0"></div>
-          </section>
-          <section id="add" class="col-6">
-            <h2>Add your score</h2>
-            <form action="/" method="GET" class="">
+        <h1 class="text-center display-2 mb-5">Leaderboard</h1>
```

Comment:
- [ ] Please fix this title in order to match the required desig
Your page
![Screenshot from 2021-10-07 22-32-01](https://user-images.githubusercontent.com/62705947/136458156-a36d620e-0408-49a8-94ec-9aeb225d8ff4.png)
Original
![Screenshot from 2021-10-07 22-33-34](https://user-images.githubusercontent.com/62705947/136458190-618b4c2a-705e-43f6-977e-fc7466c79223.png)